### DISCUSSION TITLE
how to define any extra properties related to the route in jsx?

### DISCUSSION THREAD

**Discussion by itancc** [posva/unplugin-vue-router#307] [Questions and Help] [answered]
how to define any extra properties related to the route in jsx? 
use definePage? It doesn't seem to be working

**Comment by posva** [comment, marked as answer]
yes, it's `definePage()`. You need to make sure that the jsx extension is included for pages

**Reply by itancc** [reply]
#314 Oh, that's what I wanted to ask. Thank you for your answer.
